src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx

```diff
@@ -48,6 +43,15 @@ dd {
 }
 `
 
+const mapCaseType = (type: Components.Schemas.TypeEnum) => {
+  switch (type) {
+    case "DEBRIEFING": return "Debrief"
+    case "VISIT": return "Huisbezoek(en)"
+    case "CASE": return "Aanleiding"
+    default: return ""
```

Comment:
I believe the default clause can be removed. Cause `TypeEnum` only allows the 3 options.